airflow-core/src/airflow/ui/src/i18n/locales/de/common.json

```diff
@@ -63,6 +63,11 @@
   },
   "duration": "Laufzeit",
   "endDate": "Enddatum",
+  "expand": {
+    "collapse": "Einklappen",
+    "expand": "Erweitern",
+    "tooltip": "Drücken Sie {{hotkey}}, um zu erweitern"
+  },
```

Comment:
Just fill the english translations (because those are the one we can approve) and let the other translation owners update their respective files in follow up PRs.